Update docs to account for auto-initialization (#533)
kolenaIO/kolena: docs/installing-kolena.md

## Initialization

When using `kolena`, sessions are automatically authenticated using any token present in the `KOLENA_TOKEN` environment
variable or in your `.netrc` file.

To get an API token, visit the [:kolena-developer-16: Developer](https://app.kolena.com/redirect/developer) page to
generate a token, then set the `KOLENA_TOKEN` variable in your environment:

```bash
export KOLENA_TOKEN="********"
```

By default, sessions are automatically initialized when a function requiring initialization is called. To configure

> your session, e.g. to enable extra logging via `verbose=True`, manually initialize a session by running the following:

Reviewer: ```suggestion
your session, e.g. to enable extra logging via `verbose=True`, you may call [`kolena.initialize`](./reference/initialize.md) directly:
```
nit: I like this wording is a slightly better since it encourages the user to check out the initialize options; "..running the following" is more likely to lead to copy paste without exploring imo
